src/main/resources/assets/mmorpg/lang/ru_ru.json

```
@@ -383,981 +211,204 @@
+	"mmorpg.chat.level_up_message_down": "%1$s Уровень: %2$s > %3$s!",
+	"mmorpg.chat.level_up_message_up": "Повышение уровня!",
+	"mmorpg.chat.max_map_rarity": "Эта карта имеет максимальную возможную редкость.",
+	"mmorpg.chat.max_map_rarity_for_lvl": "Эта карта имеет максимальную возможную редкость для своего уровня.",
+	"mmorpg.chat.no_charges": "У вас нет зарядов для заклинаний.",
+	"mmorpg.chat.no_duplicate_aura": "Вы не можете экипировать одинаковые самоцветы ауры.",
+	"mmorpg.chat.no_mana": "Недостаточно ресурсов для произнесения.",
+	"mmorpg.chat.none_mage": "Оружие для Не-Магов может быть использовано.",
+	"mmorpg.chat.not_in_this_dimension": "Вы не можете произнести это заклинание в этом измерении.",
+	"mmorpg.chat.not_map": "Это не карта.",
+	"mmorpg.chat.not_mns_weapon": "Вам нужно оружие Mine and Slash (Обладающее статистикой), чтобы произнести заклинания.",
+	"mmorpg.chat.open_loot_chest": "Щелкните правой кнопкой мыши, чтобы открыть ларец с добычей!",
+	"mmorpg.chat.prof_output_slot_not_empty": "Нужен хотя бы 1 пустой слот вывода.",
+	"mmorpg.chat.prof_recipe_level_not_enough": "Уровень недостаточно высок для крафта.",
+	"mmorpg.chat.prof_recipe_not_found": "Рецепт не найден",
+	"mmorpg.chat.require_mage": "Требуется магическое оружие для использования.",
+	"mmorpg.chat.require_melee": "Требуется ближнее оружие для использования.",
+	"mmorpg.chat.require_ranged": "Требуется дистанционное оружие для использования.",
+	"mmorpg.chat.soulless_gear_info": "Это снаряжение обездушено\nДуши снаряжения содержат статистику Mine and Slash\nБездушные предметы в основном бесполезны\nДуши создаются на станциях или выпадают из мобов",
+	"mmorpg.chat.spell_is_on_cd": "Заклинание перезаряжается",
+	"mmorpg.chat.using_test_spell": "Вы пытаетесь использовать тестовое заклинание вне тестовой среды",
+	"mmorpg.chat.vines_shrink": "Лианы, кажется, сжимаются, пока что...",
+	"mmorpg.chat.weapon_req_not_met": "Не выполнены требования оружия",
+	"mmorpg.chat.wrong_casting_weapon": "Этот тип оружия не может произнести это заклинание",
+	"mmorpg.chat.you_lack_jewel_slots": "Вам не хватает слотов для драгоценных камней! Вы обычно их получаете из дерева талантов. Вы можете использовать строку поиска на экране Талантов, чтобы найти их.",
 
-"_comment": " [CATEGORY]: RARITIES - NAMES",
+	"mmorpg.chat.already_max_links": "У предмета уже максимальное количество связей.",
+	"mmorpg.chat.already_max_sockets": "У предмета уже максимальное количество гнезд.",
+	"mmorpg.chat.awarded_talents": "Вы получили %1$s очков талантов.",
+	"mmorpg.chat.be_unique": "Предмет должен быть уникальным по редкости.",
+	"mmorpg.chat.can_not_go_over_maximum_level": "Нельзя превысить максимальный уровень.",
+	"mmorpg.chat.cant_go_above": "Нельзя улучшить за пределы %1$s",
+	"mmorpg.chat.cant_go_above_max_level": "Нельзя повысить уровень предмета выше максимального: %1$s",
+	"mmorpg.chat.cant_same_buff": "У вас уже есть бафф такого типа.",
```

Давай уже по-славянски раз затеяли перевод
```suggestion
	"mmorpg.chat.cant_same_buff": "У вас уже есть усиление такого типа.",
```

---

изменено